app/src/main/java/com/memory_athlete/memoryassistant/main/MainActivity.java

```diff
@@ -115,24 +140,52 @@ protected void onResume() {
         }).start();
     }
 
+    // checks if the MainActivity is running
```

Comment:
this is simply fo rthe first app start. It's purpose is to share relevant information with the user because the app is hard to use